### ISSUE TITLE
send to kindle error

### ISSUE THREAD

**Issue by perler** [janeczku/calibre-web#24]
send to kindle throws an error:

```
ERROR:cps.web:Exception on /send/2853 [GET]
Traceback (most recent call last):
  File "/opt/calibre-web/vendor/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/opt/calibre-web/vendor/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/opt/calibre-web/vendor/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/opt/calibre-web/vendor/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/opt/calibre-web/vendor/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/opt/calibre-web/vendor/flask_login.py", line 717, in decorated_view
    return func(*args, **kwargs)
  File "/opt/calibre-web/cps/web.py", line 179, in inner
    return f(*args, **kwargs)
  File "/opt/calibre-web/cps/web.py", line 611, in send_to_kindle
    result = helper.send_mail(book_id, current_user.kindle_mail)
  File "/opt/calibre-web/cps/helper.py", line 131, in send_mail
    except (socket.error, smtplib.SMTPRecipientsRefused, smtplib.SMTPException), e:
NameError: global name 'socket' is not defined
ERROR:tornado.access:500 GET /send/2853 (1.2.3.4) 1213.13ms
```

**Comment by bcg62** [comment]
also seeing this same issue

**Comment by janeczku** [comment]
Should be fixed by af1691f. Please reopen if you continue to see this issue.
